backend/routes/user.js

```diff
@@ -58,12 +57,67 @@ routes.post('/', upload, async (req, res, next) => {
     user = await registerUser(req.body, req.files || []);
   } catch (e) {
     next(e);
-    return;
   }
 
-  res.status(200).json({
+  return res.status(200).json({
     message: 'User created successfully',
     userId: user._id,
   });
 });
+
+routes.put('/update-user/:userid', async (req, res, next) => {
```

Comment:
A comment here on route naming: to be more RESTful, PUT /update-user/:userid is redundant because the PUT part already indicates that we're updating a user. There's a route now in main called PUT /:userid that updates the user, can we bring the verificationStatus check to that route?

Reply:
makes sense, have updated the same